Tasks/MavenV3/maventask.ts

```diff
@@ -565,4 +568,20 @@ function execBuildWithRestore() {
         execBuild();
     }
 }
+
+function replaceGifToBase64() {
+    try {
+        const htmlString = fs.readFileSync(path.join(reportDirectory, 'index.html'), 'utf-8');
```

Comment:
Could it make sense to put some limit for image size - to avoid issues with large images?

Reply:
I set the image size limit to 1 MB.